src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java

```diff
@@ -411,17 +441,18 @@ private MinigameTask cloneFinitequiz(final MinigameTask minigameTask, final Stri
                     minigameTask.getIndex()
                 );
             } catch (final FeignException e) {
-                if (!errorMessages.contains("finitequiz-backend not present")) {
-                    log.debug(CLONE_ERROR_MESSAGE, e);
-                    errorMessages.add("finitequiz-backend not present");
-                    return new MinigameTask(Minigame.FINITEQUIZ, "", null, minigameTask.getIndex());
-                }
+                log.debug(CLONE_ERROR_MESSAGE, e);
```

Comment:
In an exception, debug level? Also the backend name should be mentioned in the log.